R-package/NAMESPACE

```diff
@@ -0,0 +1,11 @@
+
```

Comment:
Please remove blank line at the top of the file.

Reply:
fixed and pushed changes